auth/tron-signer.go

```diff
@@ -5,16 +5,34 @@ package auth
 import (
 	"crypto/ecdsa"
 
+	"github.com/btcsuite/btcd/btcec"
```

Comment:
Ugh... so go-loom is using `github.com/btcsuite/btcd/btcec` and loomchain is using `github.com/tendermint/btcd/btcec`? That doesn't seem like a good idea, should just use `github.com/btcsuite/btcd/btcec` everywhere I suppose, in which case it shouldn't be vendored, need to pull a specific revision in the Makefile like we do with the Hashicorp plugin & go-ethereum.